DER/DerivativesContracts/Options.rdf

```diff
@@ -370,6 +560,77 @@
 		<rdfs:label xml:lang="en">floating lookback strike terms</rdfs:label>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-der-der-opt;ForeignCurrencyOption">
+		<rdfs:subClassOf rdf:resource="&fibo-der-der-opt;ExchangeTradedOption"/>
+		<rdfs:subClassOf rdf:resource="&fibo-der-fx-ctr;ForeignExchangeDerivative"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-der-opt;FxOptionUnderlying"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-dbt-ex;hasExerciseTerms"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-der-opt;CurrencyOptionExerciseTerms"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">foreign currency option</rdfs:label>
+		<skos:definition xml:lang="en">An option on a currency, based on the spot exchange rate of that currency to the currency of the security itself.</skos:definition>
```

Comment:
not sure about this - can you not have an option priced in USD on the GBP-EUR rate?